server/sonar-server/src/main/java/org/sonar/server/es/IndexingListener.java

```diff
@@ -17,11 +17,15 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-package org.sonar.server.qualityprofile.index;
+package org.sonar.server.es;
 
-import java.util.Iterator;
+import java.util.Collection;
 
-public interface ActiveRuleIterator extends Iterator<ActiveRuleDoc>, AutoCloseable {
-  @Override
-  void close();
+public interface IndexingListener {
+
+  void onSuccess(Collection<String> docIds);
+
+  static IndexingListener noop() {
+    return docIds -> {};
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00108)